app/managers.py

```diff
@@ -0,0 +1,39 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        actors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in actors_cursor]
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} VALUES (NULL, ?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def update(self, id_update: int, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} "
+            f"SET first_name = (?), last_name = (?) "
```

Comment:
What do you need brackets around `?` for?

Reply:
I thought it's a part of syntax )